src/main/java/dk/kea/kinoxp/entity/Theater.java

```diff
@@ -0,0 +1,23 @@
+package dk.kea.kinoxp.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.*;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity
+public class Theater {
+
+    @Id
+    @Column(nullable = false, length = 50, unique = true)
+    String name;
+
+    @Column(nullable = false, length = 50, unique = true)
+    int seat;
```

Comment:
Jeg er ikke sikker på at datatypen int for seats er det bedste format. Lad os bare merge det ind for nu, men lad os vende det i teamet i morgen.